activerecord/test/cases/arel/attributes/attribute_test.rb

```diff
@@ -639,6 +639,18 @@ class AttributeTest < Arel::Spec
           )
         end
 
+        if Gem::Version.new('2.6.0') <= Gem::Version.new(RUBY_VERSION)
+          it "can be constructed with a range implicitly ending at Infinity" do
+            attribute = Attribute.new nil, nil
+            node = attribute.between(0..)
```

Comment:
May I suggest this?

```suggestion
            node = attribute.between(eval("0..")) # Use eval for compatibility with Ruby < 2.6 parser.
```

This way Ruby parser in version < 2.6 will just overlook it.

Reply:
Great idea. Thanks!